engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-errata-modal.controller.js

```diff
@@ -36,7 +36,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkErrataModalC
             $scope.erratum = Erratum.get({id: errataId, 'organization_id': CurrentOrganization});
         }
 
-        nutupane = new Nutupane(HostBulkAction, {}, 'installableErrata');
+        nutupane = new Nutupane(HostBulkAction, hostIds, 'installableErrata');
```

Comment:
Yes, even though this params is named `hostIds` it's actually an object of the bulk selection type (i.e. `included: {ids: []...`).